ui/tests/acceptance/token-test.js

```diff
@@ -730,4 +730,73 @@ module('Acceptance | tokens', function (hooks) {
       requestHeaders[name.toUpperCase()]
     );
   }
+
+  module('Roles', function (hooks) {
+    // Set up a token with a role
+    hooks.beforeEach(function () {
+      window.localStorage.clear();
+      window.sessionStorage.clear();
+      faker.seed(1);
+      allScenarios.rolesTestCluster(server);
```

Comment:
This is where we call the aforementioned mirage scaffolded cluster scenario. It means that calls below using `db.tokens.findBy()`, etc. will find what they're looking for.